api/interface.py

```diff
@@ -330,6 +330,8 @@ class SoundConfig(BaseModel):
     effects: list[SoundEffect]
     """You can put as many sound effects here as you want. They stack and are added in the defined order here."""
 
+    volume: Optional[float] = 1.0
```

Comment:
Optional, so it does not need migration. Otherwise it will break when the config does not already include it.
I thought per wingman, as global volume can be controlled by the OS anyway.

Reply:
But I can change that to the default config. It does just mean, that it will break for some running from source after this.

Reply:
Made the change. Please review again :)